rust/gui-client/src-tauri/src/client/crash_handling.rs

```diff
@@ -118,7 +124,11 @@ impl minidumper::ServerHandler for Handler {
             .expect("Should be able to find logs dir to put crash dump in")
             .join("last_crash.dmp");
 
+        // `tracing` is unlikely to work inside the crash handler subprocess, so
+        // just print to stderr and it may show up on the terminal. This helps in CI / local dev.
+        eprintln!("Creating minidump at {}", dump_path.display());
         if let Some(dir) = dump_path.parent() {
+            // TODO: There is a TOCTOU bug here
             if !dir.try_exists()? {
```

Comment:
do we need to check if this exists? I think `create_dir_all` does nothing if it exists